src/Illuminate/Contracts/Cache/Repository.php

```diff
@@ -53,6 +53,24 @@ public function put($key, $value, $minutes);
     public function add($key, $value, $minutes);
 
     /**
+     * Increment the value of an item in the cache.
+     *
+     * @param  string  $key
+     * @param  mixed   $value
```

Comment:
@GrahamCampbell , would make sense to apply some fixes just before 5.3 release
See https://github.com/laravel/framework/pull/12258 https://github.com/laravel/framework/pull/10832